src/flamenco/runtime/program/fd_vote_program.c

```diff
@@ -2791,30 +2791,25 @@ fd_vote_program_execute( fd_exec_instr_ctx_t ctx ) {
    * - Feature gated, but live on mainnet.
    */
   case fd_vote_instruction_enum_authorize_checked: {
-    if( FD_LIKELY( FD_FEATURE_ACTIVE( ctx.slot_ctx, vote_stake_checked_instructions ) ) ) {
-      if( FD_UNLIKELY( ctx.instr->acct_cnt < 4 ) ) {
-        rc = FD_EXECUTOR_INSTR_ERR_NOT_ENOUGH_ACC_KEYS;
-        break;
-      }
-
-      // https://github.com/anza-xyz/agave/blob/v2.0.1/programs/vote/src/vote_processor.rs#L236
-      fd_pubkey_t const * voter_pubkey = &ctx.instr->acct_pubkeys[3];
-
-      // https://github.com/anza-xyz/agave/blob/v2.0.1/programs/vote/src/vote_processor.rs#L239
-      if( FD_UNLIKELY( !fd_instr_acc_is_signer_idx( ctx.instr, 3 ) ) ) {
-        rc = FD_EXECUTOR_INSTR_ERR_MISSING_REQUIRED_SIGNATURE;
-        break;
-      }
+    if( FD_UNLIKELY( ctx.instr->acct_cnt < 4 ) ) {
```

Comment:
nit: spacing around 4

Reply:
going to leave vote program syntax as is